.rubocop.yml

```diff
@@ -235,7 +235,7 @@ Style/RescueModifier:
 Style/SelfAssignment:
   Enabled: true
 Style/ClassAndModuleChildren:
-  Enabled: true
+  Enabled: false
```

Comment:
Not a fan of this? This is kind of a strange one because of how things worked in the past but I am fine with either.

Reply:
Yeah, this is just because I prefer the way it looks when defining a nested controller or something
```rb
class NameSpace::SomeController < ApplicationController
end

# vs
class NameSpace
  class SomeController
  end
end
```
But I'm willing to be swayed. I also meant to ask about the I18n cops and if we want to turn those off since I don't think that's going to be a concern for us given the scope of this project.

Reply:
ahh yeah, I'm fine with either so this change is fine. Also yes, we should turn that cop off.